DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/AddNewCentreCourse/SelectCourseViewModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.AddNewCentreCourse
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using Microsoft.AspNetCore.Mvc.Rendering;
+
+    public class SelectCourseViewModel
+    {
+        public SelectCourseViewModel() { }
+
+        public SelectCourseViewModel(
+            IEnumerable<SelectListItem> courseOptions,
+            IEnumerable<SelectListItem>? topicOptions,
+            int? topicId
+        )
+        {
+            CourseOptions = courseOptions;
+            TopicOptions = topicOptions;
+            TopicId = topicId;
+        }
+
+        public SelectCourseViewModel(
+            int applicationId,
+            IEnumerable<SelectListItem> courseOptions
+        )
+        {
+            ApplicationId = applicationId;
+            CourseOptions = courseOptions;
+        }
+
+        [Required(ErrorMessage = "Select a course")]
+        public int? ApplicationId { get; set; }

```

Comment:
Kept it as ApplicationId to be very clear that is what the user is selecting. The word "course" can be a bit confusing because sometimes it refers to Application, sometimes Customisation. But the user wouldn't know what we were talking about if we said "Select an application", right?

Reply:
Fair enough, but it might be worth asking about